client/src/components/ImageUploader/index.js

```diff
@@ -41,6 +41,7 @@ class ImageUploader extends Component {
         accept="image/*"
         name="avatar"
         filename={this.props.filename}
+        metadata={{ cacheControl: 'max-age=3600' }}
```

Comment:
I think it would be better to increase max-age to 1 day.

Reply:
Yes, completely agree. I am going to change it.